uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -218,6 +218,28 @@ define([
       };
 
       /**
+       * Deletes the leave request
+       */
+      this.deleteLeaveRequest = function () {
+        dialog.open({
+          title: 'Confirm Deletion?',
+          copyCancel: 'Cancel',
+          copyConfirm: 'Confirm',
+          classConfirm: 'btn-danger',
+          msg: 'This cannot be undone',
+          onConfirm: function () {
+            this.directiveOptions.leaveRequest.delete()
+              .then(function () {
+                this.cancel();
```

Comment:
please rename this mehod, `cancel` is confusing. Call it `dismissModal` or something like that